backend/parsing/ParseResult.java

```diff
@@ -0,0 +1,22 @@
+package parsing;
```

Comment:
This ParseResult may need rethinking when we parse other attributes of a submission, but this looks OK for now.